fairseq2n/python/src/fairseq2n/bindings/data/data_pipeline.cc

```diff
@@ -358,8 +358,35 @@ def_data_pipeline(py::module_ &data_module)
                 return data_pipeline::count(start, std::move(key));
             },
             py::arg("start") = 0,
-            py::arg("key") = std::nullopt);
+            py::arg("key") = std::nullopt)
+        .def_static(
+            "cat",
+            [](
+                std::vector<std::reference_wrapper<data_pipeline>> &refs1,
```

Comment:
Is there a specific reason why we take two lists of data pipelines (i.e. `refs1` and `refs2`) instead of just one?

Reply:
I was unsure how it should be implemented, but I will change it to just one